src/main/java/org/example/services/JobRoleService.java

```diff
@@ -21,15 +21,25 @@ public JobRoleService(final JobRoleDao jobRoleDao,
         this.jobRoleMapper = jobRoleMapper;
     }
 
-    public List<JobRoleResponse> getAllJobRoles() throws SQLException {
-
-        return jobRoleMapper.mapJobRoleListToJobRoleResponseList(
-                jobRoleDao.getJobRoles());
+    public List<JobRoleResponse> getAllJobRoles(
```

Comment:
it looks like we have not validated our input either at the controller level or here in the service.  

It is essential that all input data to our Rest API is validated...

Reply:
Resolved